### ISSUE TITLE
Washington Post translator gives mangled authors

### ISSUE THREAD

**Issue by kaldari** [zotero/translators#2205]
It looks like the Washington Post translator needs to be updated.

Some examples...

https://www.washingtonpost.com/national-security/2020/06/11/pentagons-top-general-apologizes-appearing-alongside-trump-lafayette-square/
author: Pentagon, Dan LamothecloseDan LamotheReporter covering the
author: militaryEmailEmailBioBioFollowFollow, the U. S.

https://www.washingtonpost.com/world/the_americas/coronavirus-brazil-bolsonaro-tests-positive/2020/07/07/5fa71548-c049-11ea-b4f6-cb39cd8940fb_story.html
author: issuesEmailEmailBioBioFollowFollow, Terrence McCoycloseTerrence McCoyReporter covering social

https://www.washingtonpost.com/dc-md-va/2020/06/07/dc-black-lives-matter-defund-police/
author: Tan, Rebecca
author: Miller, closeRebecca TanLocal reporterEmailEmailBioBioFollowFollowMichael E.
author: immigration, closeMichael E. MillerLocal enterprise reporter covering
author: poverty
author: inequality
author: Chason, crimeEmailEmailBioBioFollowFollowRachel
author: Schmidt, closeRachel ChasonReporter covering local politicsEmailEmailBioBioFollowFollowSamantha
author: gender, closeSamantha SchmidtReporter covering
author: deskEmailEmailBioBioFollowFollow, family issuesEmailEmailBioBioFollowFollowTeddy AmenabarcloseTeddy AmenabarAudience editor embedded on the Local

**Comment by kaldari** [comment]
Here's the HTML code they are currently using for authors...
Single author:
```
<div class="author-names">
	<span>By </span>
	<div class="relative dib">
		<div>
			<span data-qa="author-name-wrapper"><a class="author-name-link link-hover-underline" href="https://www.washingtonpost.com/people/terrence-mccoy/"><span class="author-name font-bold link blue hover-blue-hover">Terrence McCoy</span></a></span>
		</div>
		<div class="absolute bg-white brad-4 z-2 tooltip tooltip-bottom mt-sm dn hidden author-tooltip">
			...
		</div>
	</div>
</div>
```
Multiple authors:
```
<div class="author-names">
	<span>By </span>
	<div class="relative dib">
		<div>
			<span data-qa="author-name-wrapper"><a class="author-name-link link-hover-underline" href="https://www.washingtonpost.com/people/rebecca-tan/"><span class="author-name font-bold link blue hover-blue-hover">Rebecca Tan</span></a><span class="gray-dark">,<!-- -->&nbsp;</span></span>
		</div>
		<div class="absolute bg-white brad-4 z-2 tooltip tooltip-bottom mt-sm dn hidden author-tooltip">
			...
		</div>
	</div>
	<div class="relative dib">
		<div>
			<span data-qa="author-name-wrapper"><a class="author-name-link link-hover-underline" href="https://www.washingtonpost.com/people/michael-e-miller/"><span class="author-name font-bold link blue hover-blue-hover">Michael E. Miller</span></a><span class="gray-dark">,<!-- -->&nbsp;</span></span>
		</div>
		<div class="absolute bg-white brad-4 z-2 tooltip tooltip-top mb-sm dn hidden author-tooltip">
			...
		</div>
	</div>
</div>

**Comment by kaldari** [comment]
@adam3smith @simonster - Just wanted to bring your attention to this bug as it's causing issues with Washington Post citations on Wikipedia (https://phabricator.wikimedia.org/T255176).

**Comment by dstillman** [comment]
Needs more testing, but I have a fix in #2210.
